test/api/controllers/adminController.test.js

```diff
@@ -52,15 +48,16 @@ describe('Test: admin controller', () => {
         .catch(error => done(error));
     });
 
-    it('should raise an error if database does not exist', done => {
+    it('should raise an error if database does not exist', () => {
       request.input.args.database = 'city17';
+      adminController.throw = sinon.spy();
 
       try {
         adminController.resetCache(request);
-        done(new Error('Should not resolves'));
```

Comment:
I disagree with this removal (albeit the typo should be fixed): without it, this test passes if the admin controller does not throw.